usaspending_api/settings.py

```diff
@@ -82,12 +82,15 @@
 ES_HOSTNAME = ""
 if not ES_HOSTNAME:
     ES_HOSTNAME = os.environ.get("ES_HOSTNAME")
-TRANSACTIONS_INDEX_ROOT = os.environ.get("ES_TRX_ROOT") or "future-transactions"
+ES_TRANSACTIONS_ETL_VIEW_NAME = "transaction_delta_view"
+ES_TRANSACTIONS_MAX_RESULT_WINDOW = 50000
+ES_TRANSACTIONS_NAME_SUFFIX = "transactions"
+ES_TRANSACTIONS_READ_ALIAS_PREFIX = "transaction-records"
```

Comment:
Because ES allows wildcard searches, I thought it prudent to differentiate the index names and the alias.
`transaction-records*` won't pickup the underlying index name `*transactions` (or the sometimes previously  occurring `transactions*` index names)

Also, a search on `transactions*` won't hit the aliases.

Using `transactions-read*` could work, but there is an overlap of "transactions" which could lead to incorrect results if devs aren't careful and aware of all the indexes in the cluster.